.github/workflows/ci.yml

```diff
@@ -240,8 +200,47 @@ jobs:
       - name: Run the unit tests
         run: invoke ci-tests -m "unit" --xdist --slowest=10 --timeout=1.5
 
+  doc-checks:
+    needs: [unit-tests, static-analysis]
```

Comment:
The change here is moving this block down and updating the following 2 fields:
`needs` and `if`